cms/envs/common.py

```diff
@@ -84,6 +84,9 @@
     # django-debug-toolbar
     DEBUG_TOOLBAR_PATCH_SETTINGS,
     BLOCK_STRUCTURES_SETTINGS,
+
+    # Default currency, DOES NOT AFFECT VERIFIED STUDENTS
```

Comment:
Let's clarify this comment. What do you mean when you say this doesn't affect verified students?